launch_ros/launch_ros/utilities/to_parameters_list.py

```diff
@@ -28,25 +29,69 @@
 from ..parameters_type import EvaluatedParameters
 
 
+def __format_dict(dictionary, keys, formatted_dict):
+    """
+    Combine namespaces and their node names into one key and remove the "ros__parameters" key.
+
+    Return an empty dict if the "ros__parameters" key is not found.
+    """
+    for key, value in dictionary.items():
+        if key == 'ros__parameters':
+            node_name = '/'.join(keys)
+            formatted_dict[node_name] = value
+            return formatted_dict
+        elif type(value) is dict:
```

Comment:
do we need an `else` clause?
what should happen if `type(value)` isn't dict.

Reply:
```suggestion
        elif isinstance(value, dict):
```

Reply:
See 9de0767

I don't think an `else` clause is necessary. If we get a value that isn't a dictionary, I think it is safe to ignore it, e.g.

```
foo: 42
bar:
  ros__parameters:
    ...
```